<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>termoAIRES</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" rowspan="5">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>740</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>740</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1" rowspan="5">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(34, 83, 87);</string>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>301</width>
           <height>641</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="15" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label13">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_29">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>T05 =</string>
            </property>
           </widget>
          </item>
          <item row="15" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_13">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Super-Aquecimento = </string>
            </property>
           </widget>
          </item>
          <item row="16" column="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>kJ/kg</string>
            </property>
           </widget>
          </item>
          <item row="14" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>K</string>
            </property>
           </widget>
          </item>
          <item row="23" column="1" alignment="Qt::AlignHCenter">
           <widget class="QCheckBox" name="checkBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="17" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label15">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label02">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="16" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_15">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>h1 = </string>
            </property>
           </widget>
          </item>
          <item row="10" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label10">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="18" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_8">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>T_sat_Evap = </string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_27">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>T03 = </string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_28">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>T04 = </string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label04">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="18" column="2">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Â° C</string>
            </property>
           </widget>
          </item>
          <item row="23" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_7">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Perda Desprezivel = </string>
            </property>
           </widget>
          </item>
          <item row="20" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="13" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_12">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Q_evap = </string>
            </property>
           </widget>
          </item>
          <item row="7" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_31">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>T07 = </string>
            </property>
           </widget>
          </item>
          <item row="22" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_11">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>m =</string>
            </property>
           </widget>
          </item>
          <item row="19" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_19">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>T_sat_Cond = </string>
            </property>
           </widget>
          </item>
          <item row="22" column="2">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>kg/h</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label03">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="22" column="1">
           <widget class="QLineEdit" name="lineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(34, 83, 87);</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="14" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label12">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="18" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label16">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="19" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label17">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="16" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label14">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_33">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>P01 = </string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label05">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="17" column="2">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>kJ/kg</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>160</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="15" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>K</string>
            </property>
           </widget>
          </item>
          <item row="14" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_9">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Sub-Resfriamento = </string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label06">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="19" column="2">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Â° C</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label09">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_30">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>T06 = </string>
            </property>
           </widget>
          </item>
          <item row="10" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_34">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>P02 = </string>
            </property>
           </widget>
          </item>
          <item row="24" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label07">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_32">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>T08 = </string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_25">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>T01 = </string>
            </property>
           </widget>
          </item>
          <item row="17" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_17">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>h3 = </string>
            </property>
           </widget>
          </item>
          <item row="13" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label11">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label08">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="13" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>kW</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_26">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>T02 =</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label01">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_45">
            <property name="text">
             <string>Â° C</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_46">
            <property name="text">
             <string>Â° C</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_47">
            <property name="text">
             <string>Â° C</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLabel" name="label_50">
            <property name="text">
             <string>Â° C</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_48">
            <property name="text">
             <string>Â° C</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLabel" name="label_49">
            <property name="text">
             <string>Â° C</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QLabel" name="label_52">
            <property name="text">
             <string>Â° C</string>
            </property>
           </widget>
          </item>
          <item row="9" column="2">
           <widget class="QLabel" name="label_53">
            <property name="text">
             <string>bar</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QLabel" name="label_51">
            <property name="text">
             <string>Â° C</string>
            </property>
           </widget>
          </item>
          <item row="10" column="2">
           <widget class="QLabel" name="label_54">
            <property name="text">
             <string>bar</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="label_14">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Q_evap = </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QLabel" name="label18">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="12" column="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>BTU/h</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
